hathor/transaction/transaction.py

```diff
@@ -260,6 +263,33 @@ def to_json(self, decode_script: bool = False, include_metadata: bool = False) -
         json['tokens'] = [h.hex() for h in self.tokens]
         return json
 
+    def verify_basic(self, skip_block_weight_verification: bool = False) -> None:
```

Comment:
I guess we should add a docstring explaining what a basic validation means for a transaction.

Reply:
Done.